### DISCUSSION TITLE
Hide some domains in "Top queried domains"

### DISCUSSION THREAD

**Discussion by Dan-Sun** [AdguardTeam/AdGuardHome#5366] [Q&A] [answered]
Having a lot of requests for certain domains which fill the "Top queried domains" list with known domains. Is it possible to hide them to see actually the list without the known domains?

**Comment by ammnt** [comment, marked as answer]
> Having a lot of requests for certain domains which fill the "Top queried domains" list with known domains. Is it possible to hide them to see actually the list without the known domains?

Unfortunately no, not yet.

**Reply by Dan-Sun** [reply]
Do you know is there already an issue related to this?

**Reply by ammnt** [reply]
> Do you know is there already an issue related to this?

I don't think so. Better make a feature request😉

**Comment by devipasigner** [comment]
Under dns settings at the very bottom at Disallowed domains you can add domains which you want to be hidden and blocked. I don’t think you can hide domains without it being blocked too.

**Reply by Dan-Sun** [reply]
Yeah, I dont wanna block those domains.
